setup.py

```diff
@@ -55,6 +55,7 @@ def run(self):
     ),
     extras_require={
         'jwt': ('PyJWT >= 1.4.0',),
+        'cryptography': ('cryptography >= 2.0.3',),
```

Comment:
this is only used in tests right? why putting it here

Reply:
only tests. it's here because i had no idea where to put it

Reply:
if it's test-only, then you don't need it here at all; tox takes care of it

Reply:
it wasn't as far as I could see. I need to add the install step to the tox.ini?

Reply:
remove

Reply:
Needed now, for cert loading

Reply:
oh, then make it part of `jwt`